### PR TITLE
Config warning fix

### PR DESCRIPTION
**Pull request by dmageeLANL**
Don't have pav_config load empty configs for config.yaml files that don't exist and don't need to exist.  Because the logic around its label key gets wonky and prompts a warning.

### CODE REVIEW

**Comment by dmageeLANL** [comment]
OMG finally!

File: lib/pavilion/config.py [hunk 1]
```diff
@@ -33,13 +33,10 @@
 PAV_CONFIG_DIR = os.environ.get('PAV_CONFIG_DIR', None)
 
 if PAV_CONFIG_DIR is not None:
-    PAV_CONFIG_DIR = Path(PAV_CONFIG_DIR)
+    PAV_CONFIG_DIR = Path(PAV_CONFIG_DIR).resolve()
```
Comment (Paul-Ferrell) [inline lib/pavilion/config.py]: Resolve only works in python 3.5 if the path exists.

**Review by Paul-Ferrell — APPROVED** [review]

**Comment by dmageeLANL** [comment]
So.  Is it closed?

**Comment by Paul-Ferrell** [comment]
I was waiting on the unittests to pass and forgot about it.

**Pull request merged**